src/dashboard/cd-dashboard-children.vue

```diff
@@ -155,5 +158,25 @@
       padding: 0 0 0 5px;
     }
   }
+
+  @media (max-width: @screen-xs-max) {
+    .cd-dashboard-children {
+      max-width: 100%;
+
+      &__divider {
+        border-color: #CAC4CE;
+      }
+
+      &__badge {
+        width: 95px;
+        height: 100%;
+
+        &-image {
+          height: 90px;
```

Comment:
We'll have to fix the badges, it's horrible to look at, but I guess we can postpone it to later on (harmonization ticket) ? cc @DanielBrierton

Reply:
Ergh yes we do, Daniel mentioned some solution we could use the other day but I dont remember what it was

Reply:
Flexing the img while giving a size to the parent should work ?

Reply:
ahhhh gotcha

Reply:
errr yes that works but they looks funny with diff sizing .@DanielBrierton what were you sayin we could do the other day??